packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-node.vue

```diff
@@ -8,14 +8,17 @@
 		<div v-if="displayedInterventions.length > 0" class="intervention-summary">
 			<div v-for="(intervention, index) in displayedInterventions" :key="index">
 				<p class="name">{{ intervention.name }}</p>
+				<div>
+					Starting at time step
+					<span class="semi-bold">{{ intervention.staticInterventions[0].timestep }},</span>
+				</div>
```

Comment:
This would assume the interventions are in time order?

Reply:
So the question to ask is whether we want to pivot by starting time or not, for example I can do this and "starting time = 10" is repeated.

<img width="362" alt="image" src="https://github.com/user-attachments/assets/5f10b9cb-4924-48cd-8cb9-eec23be29729" />